analysis.json

```
@@ -1,5 +1,26 @@
 {
-  "dependencies": {
-  },
-  "sha256sum": "ca3d163bab055381827226140568f3bef7eaac187cebd76878e0b63e9e442356"
+  "vulnerabilities": {
+    "CVE-2022-36033": {
+      "dependency": {
+        "name": "org.jsoup:jsoup",
+        "affectedVersions": "[,1.15.3)"
```

Here is the case of one element in the array, I guess software reading this should consider both cases, one element represented as a string like in here, and the case of an array of strings.

---

It should always be an array for a consistent mapping to POJOs.
```suggestion
        "affectedVersions": [ "[,1.15.3)" ]
```